### ISSUE TITLE
The demo trainnig doesn't converge :(

### ISSUE THREAD

**Issue by sunbuny** [BerkeleyAutomation/gqcnn#7]
Hi～
I  am running your wiki's trainnig.py demo. But i found the trainning doesn't converge. It is so sad .
I am new to the tensorflow.
![training_curve](https://user-images.githubusercontent.com/9444218/28240265-190cf6d0-69b1-11e7-8bd8-46f082786c46.jpg)

**Comment by visatish** [comment]
Could you also post the output of Tensorboard here? Also, did you make any hyper-parameter modifications in the yaml configuration files? Also, could you specify which configuration yaml file and which dataset are being trained.

**Comment by sunbuny** [comment]
sorry.
Here is the training.yaml. I changed the dataset_dir and output_dir and model_dir. That is all.
![image](https://user-images.githubusercontent.com/9444218/28244996-f4f99834-6a2d-11e7-9948-8e890831a365.png)
I dont know how to change the hyper-parameter......
The yaml is traning.yaml and the data set is adv_synth.zip

**Comment by sunbuny** [comment]
[events.out.tfevents.1500121517.sun-PC.zip](https://github.com/BerkeleyAutomation/gqcnn/files/1150856/events.out.tfevents.1500121517.sun-PC.zip)

**Comment by jeffmahler** [comment]
There were some erroneous parameters in the cfg/tools/training.yaml on master branch. Please do a pull from master and try again. Sorry about that.

`cd /path/to/your/gqcnn`
`git pull origin master`

**Comment by jeffmahler** [comment]
Sorry one more parameter bug: dropout of the fully connected layers should be turned off. In general we recommend that you pull from master often as the package is still in beta mode and we are pushing updates often.

**Comment by sunbuny** [comment]
OK，thanks for your kindly reply.
